pipenv/routines/requirements.py

```diff
@@ -0,0 +1,78 @@
+import re
+import sys
+
+from pipenv.utils.dependencies import get_lockfile_section_using_pipfile_category
+from pipenv.vendor import click
+
+
+def requirements_from_deps(deps, include_hashes=True, include_markers=True):
+    pip_packages = []
+
+    for package_name, package_info in deps.items():
+        # Handling git repositories
+        if "git" in package_info:
+            git = package_info["git"].replace("ssh://", "https://")
```

Comment:
```suggestion
            git = package_info["git"]
```
I think this should be respected, I'm not sure why the replacement here? Anyways, I think Pipfile.lock should have the "correct" info.